### DISCUSSION TITLE
Add Project Goon, a new HRI related to the R&D server

### DISCUSSION THREAD

**Discussion by PerfectTangent** [unitystation/unitystation#8363] [Ideas]
### Description

So, /tg/ recently reworked their `download NT research notes` antag objective in https://github.com/tgstation/tgstation/pull/61685, and since @MaxIsJoe just added the R&D server machine to unitystation in #8339, It'd actually make for a good addition now to incorporate /tg/'s changes.

The way it works is the passive points generation of the R&D server is enhanced by a special hard drive inside the server (the normal passive points gain is reduced to half if the hard drive is removed). Traitors can get the objective to steal the hard drive, which is more interesting and intuitive then downloading generic data and also has a cost to the science department associated with it.

**Comment by AtnerNT** [comment]
Seems like a pretty simple addition- give R&D server objects an inventory slot that only fits the HRI, if the slot is occupied double the research rate, can be removed with a screwdriver (+ extra steps) and then just add the item to the traitor list. Honestly its simple change that I think just has a nice bit of flavor.

**Reply by AtnerNT** [reply]
Hmm, just a suggestion- but a drives purpose is to store data, if the drive is removed- perhaps some of the more advanced/expensive technologies should become locked- with insufficient data to research. So you actually feel like you are stealing sciences research from them. (Though I feel it shouldn't effect technologies that are already researched).

**Reply by AtnerNT** [reply]
in the tg pr- the action of removing the drive cannot be undone- however I feel like it should be able to- if security tracks down and takes back the drive science should be able to restore functionality. Maybe to complete the objective the drive must be burnt or destroyed

**Reply by PerfectTangent** [reply]
yeah, there are several details about the original PR that I don't think are desirable to replicate for us. /tg/ has since refined the idea a bit with follow-up PRs for themselves.

**Reply by wraith-54321** [reply]
Burning does not make much sense due to the fact that your trying to steal the drive and tech on it. Also this will have to be balanced for new research
